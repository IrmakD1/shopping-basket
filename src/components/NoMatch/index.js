import React from 'react'

const NoMatch = () => {
    return (
        <div className='no-match'>
            <h1>404</h1>
            <h2>This page does not exist</h2>
        </div>
    )
} 

export default NoMatch