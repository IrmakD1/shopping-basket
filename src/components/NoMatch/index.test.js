import React from 'react';
import renderer from 'react-test-renderer';
import NoMatch from '.';

describe('NoMatch Component', () => {
    let component;
    let tree;
  
    beforeEach(() => {
      component = renderer.create(
        <div>
                <NoMatch />
        </div>
      );
      tree = component.toJSON();
    });
  
    test('it should render', () => {
      expect(tree).toMatchSnapshot();
    });
});
  