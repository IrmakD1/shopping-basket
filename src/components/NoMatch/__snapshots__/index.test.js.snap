// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch Component it should render 1`] = `
<div>
  <div
    className="no-match"
  >
    <h1>
      404
    </h1>
    <h2>
      This page does not exist
    </h2>
  </div>
</div>
`;
